micro-rdk/src/common/registry.rs

```diff
@@ -24,6 +24,8 @@ use crate::proto::common::v1::ResourceName;
 pub enum RegistryError {
     #[error("RegistryError : Model '{0}' not found")]
     ModelNotFound(String),
```

Comment:
do we still use this error then?

Reply:
I believe so, in `get_board_constructor` etc.